lib/src/Uncertainty/Distribution/HistogramFactory.cxx

```diff
@@ -133,6 +140,63 @@ Histogram HistogramFactory::buildAsHistogram(const Sample & sample,
   return result;
 }
 
+Histogram HistogramFactory::buildAsHistogram(const Sample & sample,
+    const Scalar first,
+    const Point width) const
```

Comment:
Width should be passed by reference as it is a complex type (ie an object)

Reply:
OK